src/fc/AltFilter.cpp

```diff
@@ -84,8 +84,8 @@ void AltFilter::prefilter(SensorData& data){
   }
 
   Z(0) = p2alt(pres);  //LOL gotta convert to meters oops
-  Z(1) = (data.adxl1_data.y) - 9.807; //Antenna connector facing up, ematch connector down
```

Comment:
Perhaps this should be left in the spaceshot branch, so we have two parallel flight configs